src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeRiskLevelCellConfigurator.swift

```diff
@@ -109,8 +110,11 @@ class HomeRiskLevelCellConfigurator: HomeRiskCellConfigurator {
 		cell.bodyLabel.accessibilityIdentifier = AccessibilityIdentifiers.RiskCollectionViewCell.bodyLabel
 		cell.detectionIntervalLabel.accessibilityIdentifier = AccessibilityIdentifiers.RiskCollectionViewCell.detectionIntervalLabel
 		cell.updateButton.accessibilityIdentifier = AccessibilityIdentifiers.RiskCollectionViewCell.updateButton
-
 	}
+
+	/// Convenience method that can be overwritten to configure the button without running the full configure(_:) method.
+	/// This is handy when very frequent updates such as the update countdown are applied to the button.
+	func configureButton(for cell: RiskLevelCollectionViewCell) { }
```

Comment:
I'm currently thinking whether this method could get a better name. Is this almost like a reload view method?

Reply:
Checked with @johannesrohwer, this seems to be intrinsinc to how the rest of the home vc works. Every change has to configure the cell anew.